policy-controller/grpc/src/workload.rs

```diff
@@ -0,0 +1,87 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use std::str::FromStr;
+
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize, JsonSchema)]
+pub enum Kind {
+    #[serde(rename = "external")]
```

Comment:
probably more descriptive to call this "external_workload" in full instead of just "external" (external what?)